lib/libshare/nfs.c

```diff
@@ -149,13 +99,11 @@ get_linux_hostspec(const char *solaris_hostspec, char **plinux_hostspec)
 		 * to skip the @ in this case
 		 */
 		*plinux_hostspec = strdup(solaris_hostspec + 1);
-	} else {
+	} else
 		*plinux_hostspec = strdup(solaris_hostspec);
-	}
```

Comment:
[cstyle] The { } are expected for the `else` block when need by the `if` portion.